From 2b7193f402d79ae7ec0327512511425db29b4f5f Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Fri, 6 Jul 2018 17:11:55 +0800
Subject: [PATCH 4258/5242] MLK-18605-5 drm/bridge: sec-dsim: correct
 'DSIM_FIFOCTRL' offset

commit  4c3aa0846c084c633214cd03303e9b854bdc30c7 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the DSIM specification, the 'DSIM_FIFOCTRL'
register addr offset should be '0x4c' instead of '0x48'.
So correct it.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit 31850816f3aea109aff6c4bbcb44221e7d74afb5)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/bridge/sec-dsim.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/sec-dsim.c b/drivers/gpu/drm/bridge/sec-dsim.c
index 6752ccc..dc64a9c 100644
--- a/drivers/gpu/drm/bridge/sec-dsim.c
+++ b/drivers/gpu/drm/bridge/sec-dsim.c
@@ -52,7 +52,7 @@
 #define DSIM_PAYLOAD			0x40
 #define DSIM_RXFIFO			0x44
 #define DSIM_FIFOTHLD			0x48
-#define DSIM_FIFOCTRL			0x48
+#define DSIM_FIFOCTRL			0x4c
 #define DSIM_MEMACCHR			0x50
 #define DSIM_MULTI_PKT			0x78
 
-- 
1.7.9.5

